packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-container.tsx

```diff
@@ -0,0 +1,172 @@
+                const { get_account_status } = response;
+                if (get_account_status) {
+                    const {
+                        allow_document_upload,
+                        allow_poa_resubmission,
+                        needs_poi,
+                        needs_poa,
+                        document_status,
+                        is_age_verified,
+                        poa_address_mismatch,
+                    } = populateVerificationStatus(get_account_status);
+                    const has_submitted_poa = document_status === PoaStatusCodes.pending && !allow_poa_resubmission;
+
+                    setAuthenticationStatus({
+                        ...authentication_status,
+                        allow_document_upload,
+                        allow_poa_resubmission,
+                        needs_poi,
+                        needs_poa,
+                        document_status,
+                        has_submitted_poa,
+                        is_age_verified,
+                        poa_address_mismatch,
+                    });
+                    setIsLoading(false);
+                    refreshNotifications();
+                }
+            });
+        }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [is_switching]);
+
+    const handleResubmit = () => {
+        setAuthenticationStatus({ ...authentication_status, ...{ resubmit_poa: true } });
+    };
+
+    const onSubmit = (needs_poi: boolean) => {
+        setAuthenticationStatus({ ...authentication_status, ...{ has_submitted_poa: true, needs_poi } });
+    };
+
+    const {
+        allow_document_upload,
+        allow_poa_resubmission,
+        document_status,
+        needs_poi,
+        resubmit_poa,
+        has_submitted_poa,
+        is_age_verified,
+        poa_address_mismatch,
+    } = authentication_status;
+
+    const from_platform = getPlatformRedirect(app_routing_history);
+
+    const should_show_redirect_btn = Object.keys(platforms).includes(from_platform?.ref ?? '');
+
+    const redirect_button = should_show_redirect_btn ? (
+        <Button
+            primary
+            className='proof-of-identity__redirect'
+            onClick={() => {
+                const url = platforms[from_platform.ref as keyof typeof platforms]?.url;
+                if (url) {
+                    window.location.href = url;
+                    window.sessionStorage.removeItem('config.platform');
+                }
+            }}
+        >
+            <Localize i18n_default_text='Back to {{platform_name}}' values={{ platform_name: from_platform.name }} />
+        </Button>
+    ) : null;
+
+    if (is_loading) return <Loading is_fullscreen={false} className='account__initial-loader' />;
+    if (
+        !allow_document_upload ||
+        (!is_age_verified && !allow_poa_resubmission && document_status === 'none' && is_mx_mlt)
```

Comment:
do we need is_mx_mlt check here?

Reply:
not sure, I didn’t touch this part of code. it was implemented before